set_up.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# Ask the user for a username
```

Comment:
I haven't created many shell scripts, so this is genuinely a question rather than a comment. 

What's the logic behind using comments rather than methods? Is it more idiomatic to use comments in shell scripts?